app/web/src/newhotness/ComponentDetails.vue

```diff
@@ -55,31 +55,30 @@
       <div class="flex-none">{{ component.schemaVariantName }}</div>
       <div class="flex-none">/</div>
       <div class="flex-1 min-w-0">
-        <EditInPlace ref="editInPlaceRef" @hidden="reset" @showing="focus">
-          <template #trigger>
-            <VButton
-              :label="component.name"
-              :loading="wForm.bifrosting.value"
```

Comment:
We talked about how we don't want any spinner activity, so we can remove label and loading. And the `{{ component.name }}` needs to change, because once it flips back to the button, its not going to show the value the user typed in.